src/main/kotlin/dev/haarolean/gnutardbot/abilities/ForwardSuggestionsAbility.kt

```diff
@@ -0,0 +1,43 @@
+package dev.haarolean.gnutardbot.abilities
+
+import dev.haarolean.gnutardbot.TardBot
+import dev.haarolean.gnutardbot.util.MemberUtils
+import org.telegram.abilitybots.api.objects.MessageContext
+import org.telegram.telegrambots.meta.api.methods.ForwardMessage
+import org.telegram.telegrambots.meta.api.methods.groupadministration.GetChat
+import org.telegram.telegrambots.meta.api.objects.Message
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException
+
+class ForwardSuggestionsAbility(private val bot: TardBot) : AbilityHandler {
+    private val memberUtils: MemberUtils = (MemberUtils(bot))
+
+    override fun isApplicable(ctx: MessageContext): Boolean {
+        val update = ctx.update()
+        val message: Message = when {
+            update.hasMessage() -> update.message
+            update.hasEditedMessage() -> update.editedMessage
+            else -> return false
+        }
+        if (bot.isAdmin(message.from.id)) return false
+        if (!message.isUserMessage) return false
+        val chat = bot.execute(GetChat(bot.properties.chatId))
```

Comment:
I'd rather not call any API methods within `isApplicable` as I believe these methods should be executed quickly (as we're iterating over all the handlers to see which ones are applicable). Also, failing these requests would bring other unwanted side effects. I suggest moving these external API checks within `handle` and keeping whatever we can in `isApplicable`. What do you think?